src/components/hooks/use-open-live-chat-redirection.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { redirectOpenLiveChatBox } from 'common/utility'
 
-export const useOpenLiveChat = (props) => {
+export const useOpenLiveChat = (props?: boolean) => {
```

Comment:
how about we change `props` to `isRedirectEnabled` or something else, more meaningful name I mean?

Reply:
Have sense, changed on `is_triggered`